bindings/pyroot/pythonizations/test/numbadeclare.py

```diff
@@ -61,10 +49,7 @@ def f2(x):
         fn0 = ROOT.Numba.Declare(["float"], "float")(f1)
         ref = nb.cfunc("float32(float32)", nopython=True)(f2)
         gc.collect()
-        if sys.version_info.major == 2:
-            self.assertEqual(sys.getrefcount(f1), sys.getrefcount(f2) + 1)
-        else:
-            self.assertEqual(sys.getrefcount(f1), sys.getrefcount(f2) + 2)
+        self.assertEqual(sys.getrefcount(f1), sys.getrefcount(f2) + 2)
```

Comment:
(Not for this PR, but a general note) The Python reference count is not very predictable, it would be better to not have logic like this.

Reply:
Ouch, indeed :( Should be corrected in a separate PR